src/client/datascience/jupyter/kernels/jupyterKernelService.ts

```diff
@@ -146,6 +146,8 @@ export class JupyterKernelService {
             };
         }
 
+        traceInfo(`RegisterKernel for ${kernel.id}`);
```

Comment:
Removed most of the logging that I had to debug this, but left in a few that seemed more useful.